.github/workflows/autorelease-preview.yml

```diff
@@ -0,0 +1,55 @@
+name: Auto release preview @excalidraw/excalidraw-preview
+on:
+  issue_comment:
+    types: [created, edited]
+
+jobs:
+  Auto-release-excalidraw-preview:
+    name: Auto release preview
+    if: github.event.comment.body == '@excalibot trigger release' && github.event.issue.pull_request
+    runs-on: ubuntu-latest
+    steps:
+      - name: React to trigger comment
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          token: ${{ secrets.PUSH_TRANSLATIONS_COVERAGE_PAT }}
```

Comment:
This I think we should rename to `EXCALIBOT_GITHUB_TOKEN`

Reply:
is it even necessary? Shouldn't the [default](https://github.com/peter-evans/create-or-update-comment#action-inputs) `GITHUB_TOKEN` work when within the same repo?